omod/src/main/resources/config.xml

```diff
@@ -183,6 +183,37 @@
 		<defaultValue>100</defaultValue>
 	</globalProperty>
 
+    <globalProperty>
+        <property>ugandaemr.dsdm.allowClinicalOverrideDSDMPatientStability</property>
+        <description>When set to true, this allows to override the decision made by the system about DSDM Stability </description>
+        <defaultValue>false</defaultValue>
+        <dataType>org.openmrs.customdatatype.datatype.BooleanDatatype</dataType>
+    </globalProperty>
+
+    <globalProperty>
+        <property>ugandaemr.dsdm.currentRegimenDurationRequirementInMonths</property>
+        <description>The number of months when on current regimen required for patient to be categorized as stable</description>
+        <defaultValue>6</defaultValue>
+    </globalProperty>
+
+    <globalProperty>
```

Comment:
@slubwama Actually it is 999 since 1,000 is taken as non-supressed

Reply:
Done